source/js/buyers-guide/bg-main.js

```diff
@@ -112,6 +112,23 @@ let main = {
     if (document.querySelector(`body.pni.catalog`)) {
       HomepageSlider.init();
     }
+    if (document.querySelector("#view-article")) {
+      const firstParagraph = document.querySelector(
+        ".paragraph-block .rich-text"
+      );
+      const relatedContainer = document.querySelector("#pni-related-content");
+      if (!firstParagraph || !relatedContainer) return;
+      const relatedContent = relatedContainer.querySelector("div");
+      const relatedPlace = firstParagraph.querySelector("*:nth-child(3)");
+      if (!relatedPlace) return;
+      firstParagraph.insertBefore(relatedContent, relatedPlace);
+      relatedContent.classList.add(
+        "large:tw-float-right",
+        "tw-p-4",
+        "tw-pr-0",
+        "large:tw-mr-[-20%]"
+      );
+    }
```

Comment:
What do you think about using a named function for this so it is obvious what it does without having to read through the whole function body?

Reply:
addressed